File: typescipt/src/compiler/JavaScript/builtInFunctions/minutesFunctionCall.ts
import {EndStartDateFunctionCall} from "./endStartDateFunctionCall";
import {MinutesFunction} from "../../../language/expressions/functions/minutesFunction";

export class MinutesFunctionCall extends EndStartDateFunctionCall {
   protected override className = "BuiltInDateFunctions";
   protected override methodName = "minutes";

   constructor(functionNode: MinutesFunction) {
      super(functionNode);
   }
}

File: dotnet/Lexy/Lexy.Compiler/Compiler/CSharp/BuiltInFunctions/MinutesFunctionCall.cs
using Lexy.Compiler.Language.Expressions.Functions;
using Lexy.RunTime;

namespace Lexy.Compiler.Compiler.CSharp.BuiltInFunctions;

internal class MinutesFunctionCall : EndStartDateFunctionCall<MinutesFunction>
{
    protected override string ClassName => nameof(BuiltInDateFunctions);
    protected override string MethodName => nameof(BuiltInDateFunctions.Minutes);
}